docs/graphql/manual/mutations/upsert.rst

```diff
@@ -104,6 +104,53 @@ the columns specified in ``update_columns``:
 
 The ``published_on`` column is left unchanged as it wasn't present in ``update_columns``.
 
+Update selected columns on conflict using a filter
+--------------------------------------------------
+Insert a new object in the ``article`` table, or if the primary key constraint ``article_pkey`` is violated, update
+the columns specified in ``update_columns`` only if provided the ``where`` condition is met:
```

Comment:
Sorry, this was my fault—I told you the wrong thing. The “the” in this sentence should come before the word “provided,” so it should read “update the columns specified in `update_columns` only if the provided `where` condition is met.” I’ve made the change myself, though, so you don’t have to worry about it.